### DISCUSSION TITLE
About the merge from mapping into master

### DISCUSSION THREAD

**Discussion by gcarreno** [wp-xyz/corona#45] [General]
Hey Werner(@wp-xyz)

I was afraid this would happen.

All the changes I made to `master` while you where doing your thing on `mapping` have been overwritten when your merged `mapping` into master.
Mainly of interest are the changes I made to `cdownloadmanager.pas`. But I can replicate them since I made them to my example project and then copy/paste to corona.

I still need to compare `master` pre-merge and post-merge to see what else has been lost.

Cheers,
Gus
<hr>

PS: And I still need that license decision for me to **finally** publish a release pre-merge and then post-merge!!

**Comment by wp-xyz** [comment]
Am 21.03.2021 um 16:21 schrieb gcarreno:
>
> Hey ***@***.*** <https://github.com/wp-xyz>)
>
> I was afraid this would happen.
>
> All the changes I made to |master| while you where doing your thing on
> |mapping| have been overwritten when your merged |mapping| into master.
> Mainly of interest are the changes I made to |cdownloadmanager.pas|.
> But I can replicate them since I made them to my example project and
> then copy/paste to corona.
>
This kind of work really is a pain, sorry! I think svn is a lot easier.

I already noticed the issue and manually copied all your changes in the
download manager into master again. It should be working, but you should
double-check again.


> ------------------------------------------------------------------------
>
> PS: And I still need that license decision for me to *finally* publish
> a release pre-merge and then post-merge!!
>
Give me a little more time...
